src/gremlin.py

```diff
@@ -0,0 +1,8 @@
+import os
+from gremlin_connect.gremlin_adapter import GremlinAdapter
+
+_GREMLIN_DEFAULT_HOST = os.environ.get('GREMLIN_DEFAULT_HOST', 'localhost')
```

Comment:
I would prefer configuration be managed entirely in a single place (a config.py file inside your module, like we do for all our other projects).

Reply:
Done.